Docker.Build.psd1

```diff
@@ -68,24 +67,35 @@
     # NestedModules = @()
 
     # Functions to export from this module
-    FunctionsToExport = @("Invoke-DockerBuild, Invoke-DockerTag", "Invoke-DockerLint")
+    FunctionsToExport    = @("Invoke-DockerBuild, Invoke-DockerTag", "Invoke-DockerLint")
 
     # Cmdlets to export from this module
-    CmdletsToExport = '*'
+    CmdletsToExport      = '*'
 
     # Variables to export from this module
-    VariablesToExport = '*'
+    VariablesToExport    = '*'
 
     # Aliases to export from this module
-    AliasesToExport = '*'
+    AliasesToExport      = '*'
 
     # Private data to pass to the module specified in RootModule/ModuleToProcess. This may also contain a PSData hashtable with additional module metadata used by PowerShell.
-    PrivateData = @{
+    PrivateData          = @{
 
         PSData = @{
 
             # Tags applied to this module. These help with module discovery in online galleries.
-            Tags = @('powershell', 'core', 'jenkins', 'hudson', 'api', 'PSEdition_Core', 'Windows', 'Linux')
+            Tags       = @('powershell',
+                'core',
+                'docker',
+                'build',
+                'pipeline',
+                'test',
+                'lint', '
+                    linter',
+                'integration',
+                'PSEdition_Core',
+                'Windows',
+                'Linux')
```

Comment:
What is this formatting called? 😆

Reply:
I believe the name is: 'what happened, VSCode Code Cleanup?' :) Glad, this is still just a draft 😅